pkg/asset-manager-utils/contracts/aave/DataTypes.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: agpl-3.0
```

Comment:
We don't seem to use this file anywhere. Shall we remove it for now?

Reply:
It's used by `ILendingPool.sol` (for various reserve config arguments)

Reply:
Ah gotcha, sorry that file was folded.